galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.mq.manager/bnd.bnd

```diff
@@ -0,0 +1,22 @@
+-snapshot: ${tstamp}
+Bundle-Name: MQ Manager
+Export-Package: !dev.galasa.mq.internal*,\
+        dev.galasa.mq*,\
+        javax.jms*,\
+        javax.management*,\
+        com.ibm.msg.client.wmq*
+Import-Package: !javax.validation.constraints,\
+        dev.galasa,\
+        dev.galasa.framework.spi,\
+        dev.galasa.framework.spi.cps,\
+        dev.galasa.framework.spi.creds,\
+        dev.galasa.framework.spi.language,\
+        org.apache.commons.logging.*,\
+        javax.management,\
+        javax.naming,\
+        javax.net.ssl,\
+        javax.security.cert
+Embed-Transitive: true
+Embed-Dependency: *;scope=compile
+-includeresource: com.ibm.mq.allclient-9.2.3.0.jar; lib:=true,\
+    javax.jms-api-2.0.1.jar; lib:=true
```

Comment:
needs a new line at the end, bnd gets funny about it some times

Reply:
done